src/newViews/AnalysisModule/AnalysisEditModal/index.tsx

```diff
@@ -212,6 +213,18 @@ function AnalysisEditModal(props: AnalysisEditModalProps) {
             if (response) {
                 onSuccess(response, isDefined(analysisToEdit));
             }
+            notify.send({
+                title: _ts('analysis.editModal', isDefined(analysisToEdit)
+                    ? 'analysisEdit'
+                    : 'analysisCreate',
+                ),
+                type: notify.type.SUCCESS,
+                message: _ts('analysis.editModal', isDefined(analysisToEdit)
+                    ? 'analysisEditSuccessful'
+                    : 'analysisCreateSuccessful',
+                ),
+                duration: notify.duration.MEDIUM,
+            });
```

Comment:
Instead of a conditional inside the _ts function, creating separate notify will show the items in string management page